storage/scylla_table.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef TA_SCYLLA_TABLE_H_
+#define TA_SCYLLA_TABLE_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "accelerator/errors.h"
+#include "cassandra.h"
+#include "utarray.h"
+
+typedef struct identity_s identity_t;
```

Comment:
I am afraid of the potential conflicts with `identity_t`. Can you move it to implementation-specific files rather than header(s).

Reply:
I would like to rename it to identity_table_s and identity_table_t.
It will be used in some places including promoter and TA when successfully send transactions to IRI.